README.md

```
@@ -63,6 +64,27 @@ https://backscattering.de/web-boardimage/board.svg?fen=5r1k/1b4pp/3pB1N1/p2Pq2Q/
+#### Supported Piece Sets
+
+<details>
+<summary>Show all supported piece sets</summary>
+
+When `pieceSet` is omitted, SVGs use python-chess's built-in Burnett pieces. Passing `cburnett` selects the external `cburnett` piece-set files.
+
+`alpha`, `anarcandy`, `caliente`, `california`, `cardinal`, `cburnett`, `celtic`, `chess7`, `chessnut`, `companion`, `cooke`, `dubrovny`, `fantasy`, `fresca`, `gioco`, `governor`, `horsey`, `icpieces`, `kiwen-suwi`, `kosal`, `leipzig`, `letter`, `libra`, `maestro`, `merida`, `monarchy`, `mono`, `mpchess`, `pirouetti`, `pixel`, `reillycraig`, `riohacha`, `shapes`, `spatial`, `staunty`, `tatiana`, `random`
+
+</details>
+
+### `GET /piece.png` render a PNG
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)